build

```diff
@@ -2,6 +2,11 @@
 
 set -e -u
 
+if ! [ -d ./image ] && ! [ -d ./rootfs ]; then
```

Comment:
This brings up an interesting point - perhaps it's valid to not configure *either* if you just want to see if an image is able to build at all? :thinking: